build_tools/python/e2e_test_framework/models/README.md

```diff
@@ -10,6 +10,7 @@ This directory contains the definitions of source models for e2e tests
     -   You can ask IREE team members for help if you don't have access.
 2.  Register a unique model ID in
     [build_tools/python/e2e_test_framework/unique_ids.py](/build_tools/python/e2e_test_framework/unique_ids.py).
+    -   Run uuid.uuid4() to generate a new uuid for the new model.
```

Comment:
Note that you can also run `uuidgen` on Linux. We should probably explain more explicitly that these IDs are totally arbitrary artificial keys.

Reply:
Done.